seatunnel-connectors/seatunnel-connector-spark-fake/src/main/scala/org/apache/seatunnel/spark/source/FakeStream.scala

```diff
@@ -56,7 +56,11 @@ class FakeStream extends SparkStreamingSource[String] {
   }
 
   override def checkConfig(): CheckResult = {
-    checkAllExists(config, "content")
+    if (config.hasPath("content") && config.getStringList("content").nonEmpty) {
+      CheckResult.success()
```

Comment:
Maybe we can check if the type of `config.getAnyRef("content")`  is` List`  in `CheckConfigUtil.checkAllExists()`, which can be in common use.

Reply:
@simon824  what do you think ?

Reply:
Actually, this problem exist in many other connectors, it's unnecessary to fix all connectors `checkConfig` one by one.  
#1214 this pr has fix similar problem.

Reply:
this pr seems not bad, it's suitable for this issue, but put all checks to CheckConfigUtil.checkAllExists() may cause lacking of type check, for example, if you want to get a intList, but provide stringList.
If you get a good idea about how to check all param's existance along with it's type, please tell me, thanks.